src/Statement.js

```diff
@@ -321,7 +339,9 @@ TinCan client library
                     "context",
                     "authority"
                 ],
-                i;
+                a,
+                i,
+                temp;
```

Comment:
Alternatively, always set it (as a better name) and drop the `.push` redundancy.